osu.Game/Screens/Play/HUD/GameplayScoreCounter.cs

```diff
@@ -42,7 +44,8 @@ private void load(OsuConfigManager config, ScoreProcessor scoreProcessor)
                 }
             }, true);
 
-            Current.BindTo(scoreProcessor.TotalScore);
+            totalScoreBindable = scoreProcessor.TotalScore.GetBoundCopy();
+            totalScoreBindable.BindValueChanged(_ => Current.Value = scoreProcessor.GetDisplayScore(scoreDisplayMode.Value), true);
```

Comment:
This feels a bit weird (using one bindable's `BindValueChanged` but then retrieving a completely different method).

Can you remind me what the reasoning was behind removing `Mode` from `ScoreProcessor`? I think it's one of the areas we discussed but I can't recall why this change was required.

Reply:
Because `ScoreProcessor` should never expose a non-standardised total score. Classic score is purely for display purposes.